service/migration_manager.cc

```diff
@@ -644,381 +644,118 @@ public void notifyDropAggregate(UDAggregate udf)
 }
 #endif
 
-std::vector<mutation> prepare_keyspace_update_announcement(replica::database& db, lw_shared_ptr<keyspace_metadata> ksm, api::timestamp_type ts) {
```

Comment:
> If you give me a good reason for this movement (better than "their placements in .hh and .cc have been mismatched" -- I don't really understand why this would be a problem) then I won't complain.

I don't have a better reason. I've removed this commit.

> I don't think this commit is necessary, in fact it has a drawback: it will be harder to find the authors of logical changes in this code through git blame.

Right, I didn't think about it. I'll keep that in mind in the future.